test/rates_test.rb

```diff
@@ -221,6 +231,29 @@ def test_invalid_currency_code_fails_with_rate_not_found_error
     end
   end
 
+  def test_exchange_rate_set_not_cached
+    cache = OpenExchangeRates::Cache::MemoryCache.new
+    stub(cache).get { nil }
+    stub(cache).set { open_asset('2012-05-10.json') }
+
+    fx = OpenExchangeRates::Rates.new(cache: { type: :memory })
+    fx.cache = cache
+
+    fx.parse_on("2012-05-10")
+    assert_received(cache) { |o| o.set('open-exchange-rates:2012-05-10', anything) }
```

Comment:
Line is too long. [84/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.